weave/trace_server/migrations/007_stats_views.down.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS weave_trace_db.calls_merged_stats;
+DROP VIEW IF EXISTS weave_trace_db.calls_merged_stats_view;
+
+DROP TABLE IF EXISTS weave_trace_db.object_versions_stats;
+DROP VIEW IF EXISTS weave_trace_db.object_versions_stats_view;
+
+DROP TABLE IF EXISTS weave_trace_db.table_rows_stats;
+DROP VIEW IF EXISTS weave_trace_db.table_rows_stats_view;
+
+DROP TABLE IF EXISTS weave_trace_db.files_stats;
+DROP VIEW IF EXISTS weave_trace_db.files_stats_view;
```

Comment:
Note: you don't actually need the `weave_trace_db` prefix here, or anywhere as the DB is a param of the entire migration process

Reply:
fixed